### DISCUSSION TITLE
Could not find a declaration file for module '@xstate/react'

### DISCUSSION THREAD

**Discussion by RoyalHunt** [statelyai/xstate#2997] [Q&A]
Hi there!
I've tried to update xstate library to the latest version. But start receiving the error on the build process. Do you have any idea where I can take a look to fix that? 
<img width="963" alt="Screenshot 2022-01-31 at 23 33 59" src="https://user-images.githubusercontent.com/22590130/151883181-2314cc26-941f-4240-bb79-bebaa867d4eb.png">

Thank you!

**Comment by johnlombardo-dev** [comment]
I would normally get an error like this if something has interrupted my `npm install`. The simplest fix is to delete `/node_modules`, run `npm cache clean`, and then run `npm install` again.
